### PR TITLE
lesson-2

### PR DESCRIPTION
**Pull request by DariaRomanowska**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1 +1,49 @@
...
+console.log(student);
+
+user = student; //переопределены данные на 'Daria Ramanowska'
+console.log(user);
+
+let test = 1;
+test++;
+
+console.log(test + '1'); // '21' string
+console.log(test - 1); // 1 number
```
Comment (5tronciy) [inline main.js]: Wrong comment

File: main.js [hunk 2]
```diff
@@ -1 +1,49 @@
...
+let user;
+user = 'John Doe';
+console.log(user);
+
+let student;
+// eslint-disable-next-line prefer-const
+student = 'Daria Ramanowska';
+console.log(student);
+
+user = student; //переопределены данные на 'Daria Ramanowska'
```
Comment (5tronciy) [inline main.js]: Write the comments on English only.

File: main.js [hunk 3]
```diff
@@ -1 +1,49 @@
...
+console.log(user);
+
+let test = 1;
+test++;
+
+console.log(test + '1'); // '21' string
+console.log(test - 1); // 1 number
+console.log(test === 3); //false
```
Comment (5tronciy) [inline main.js]: It doesn't assign with the task requirement. You should convert the value of variable `test` to a boolean.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Not all requirements are met. And try not to load the files that are not applied to the current task (lesson.js here).

File: main.js (hunk 1 above)
Comment (DariaRomanowska) [reply main.js]: why?

Comment (5tronciy) [reply main.js]: Oh, I didn't mention that you haven't changed the variable test. 
I think that it doesn't assign the task in this case. Make the necessary changes, please.

File: main.js [hunk 4]
```diff
@@ -7,21 +7,22 @@ let student;
...
 
 let test = 1;
 test++;
 
-console.log(test + '1'); // '21' string
+console.log(test + "1"); // '21' string
 console.log(test - 1); // 1 number
-console.log(test === 3); //false
+
+let newTest = Boolean(false);
```
Comment (5tronciy) [inline main.js]: Where is the converting of variable `test`? This statement doesn't include it.

File: main.js [hunk 5]
```diff
@@ -7,21 +7,22 @@ let student;
...
 
-user = student; //переопределены данные на 'Daria Ramanowska'
+user = student; //redefined data to 'Daria Ramanowska'
 console.log(user);
 
 let test = 1;
 test++;
 
-console.log(test + '1'); // '21' string
+console.log(test + "1"); // '21' string
```
Comment (5tronciy) [inline main.js]: It is better to use a consistent style for the strings. I think prettier may solve it.

File: main.js [hunk 6]
```diff
@@ -33,17 +34,17 @@ console.log('----------');
 const arr2 = [2, 5, 8, 15, 0, 6, 20, 3];
 
 for (const i of arr2) {
-  if (i > 5 && i < 10) {
+  if (i >= 5 && i <= 10) {
```
Comment (5tronciy) [inline main.js]: It doesn't assign the task now.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Be careful with the task.

File: main.js [hunk 7]
```diff
@@ -1 +1,55 @@
...
+let test = 1;
+test++;
+
+test = test + '1';
+console.log(test); // '21' string
+
+test = test - 1;
+console.log(test); // 1 number
+
+test = Boolean(false);
```
Comment (5tronciy) [inline main.js]: Why false?
Look at previous operations with `test`. It should be similar.

File: main.js [hunk 8]
```diff
@@ -1 +1,55 @@
...
+for (const elem of arr2) {
+  if (elem % 3) {
+    console.log(elem);
+  }
+}
+
+
+
+
+
```
Comment (5tronciy) [inline main.js]: Don't push empty rows. Do you use prettier?

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Need improvement.

File: main.js [hunk 9]
```diff
@@ -1 +1,50 @@
...
+test = test + '1';
+console.log(test); // '21' string
+
+test = test - 1;
+console.log(test); // 20 number
+
+
+console.log(test === 20);
+
```
Comment (5tronciy) [inline main.js:24]: The requirement of transforming 'test' to boolean type doesn't meet.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
The task requirement is not done.

File: main.js [hunk 10]
```diff
@@ -1 +1,50 @@
...
+let test = 1;
+test++;
+
+test = test + '1';
+console.log(test); // '21' string
+
+test = test - 1;
+console.log(test); // 20 number
+
+test = Boolean(test === 20);
```
Comment (5tronciy) [inline main.js]: What is the magic number 20 for here? Where did you get it?
It doesn't match the task.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Transform variable 'test' to boolean type.

**Review by 5tronciy — APPROVED** [review]
Bingo!
Approve.

**Comment by 5tronciy** [comment]
After approval and only after it, you may merge PR yourself. Let me know if something goes wrong with it.

**Pull request merged**